acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -5,24 +5,36 @@
 require 'openssl'
 require 'socket'
 require 'json'
+require 'securerandom'
 
 # This file contains general test helper methods for pxp-agent acceptance tests
 
 # The standard path for git checkouts is where pcp-broker will be
 GIT_CLONE_FOLDER = '/opt/puppet-git-repos'
 
 # Some helpers for working with the log file
-PXP_LOG_FILE_CYGPATH = '/cygdrive/c/ProgramData/PuppetLabs/pxp-agent/var/log/pxp-agent.log'
-PXP_LOG_FILE_POSIX = '/var/log/puppetlabs/pxp-agent/pxp-agent.log'
+PXP_LOG_FILE_CYGPATH = '/cygdrive/c/ProgramData/PuppetLabs/pxp-agent/var/log'
```

Comment:
<nitpick>These variable names should now be PXP_LOG_DIR_POSIX or PXP_LOG_PATH_POSIX. The FILE in the variable name was to indicate that they point to the file itself</nitpick>